.github/workflows/check.yml

```diff
@@ -0,0 +1,39 @@
+name: Check
+
+on:
+  push:
+    branches:
+      - master
+  pull_request: {}
+
+jobs:
+  test:
+    name: Test
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Set up Ruby
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: '3.4'
```

Comment:
Sounds great!

My only suggestion is given it's runtime which is, in theory, compatible with multiple Ruby versions, it makes sense to test it against a matrix of Ruby versions.

Reply:
> My only suggestion is given it's runtime which is, in theory, compatible with multiple Ruby versions, it makes sense to test it against a matrix of Ruby versions.

Makes sense - since https://github.com/kaitai-io/kaitai_struct_ruby_runtime/pull/13/commits/402802207fe878e45c311bfdebaedf061dabeb99, we test on Ruby 3.4, 2.4 and 1.9.